lib/tenant-management/lambdas/onboard-tenant.ts

```diff
@@ -0,0 +1,114 @@
+import { captureAWS } from "aws-xray-sdk";
```

Comment:
The idea is to trigger that lambda manually, when new tenants join?

Reply:
Yes, looks like it to me.
@cremich Would maybe be good to place an examplary invoke command in the readme for this and also have this lambdas ID in the stack outputs

Reply:
Also, some validation of the input event might be good, as I tried this:
```
aws lambda invoke --function-name dev-kiko-app-tenantmanagementonboardtenantfunction-Wj7q1XZEJtst --cli-binary-format raw-in-base64-out --payload '{"poolNames": ["First Pool", "Second Pool"], "tenantName": "Kita 123", "tenantEmail": "my_email@example.de"}' log.json
```
and the cognito group failed to create due to the space in the name.
At that point where it fails, the other resources are already successfully created and so I cannot rerun the script with the fixed tenantName as it will complain that the user already exists now.